src/Qowaiv/Qowaiv.csproj

```diff
@@ -5,10 +5,12 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Version>7.1.1</Version>
+    <Version>7.1.2</Version>
```

Comment:
I'm not sure if there is any hurry in shipping this one. We shipped 7.1.1 yesterday containing the same code basically.

Reply:
Reverted this